_im-plugin/ism/error-prevention/index.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: ISM error prevention
+parent: Index State Management
+nav_order: 1
+has_children: true
+redirect_from:
+  - /im-plugin/ism/error-prevention/
+---
+
+# ISM error prevention
+
+Error prevention validates ISM actions before they are performed to avoid actions from failing. It also discloses more information from the action validation results in the response of the [Index Explain API]({{site.url}}{{site.baseurl}}/im-plugin/ism/api/#explain-index). Validation rules and troubleshooting of each action are listed below.
```

Comment:
"... discloses more information ..." isn't clear to me. Discloses sounds like something is hidden in the results of one API and this second API exposes it.
Is the API providing more detailed information than what you get in the Index Explain API results? Is it elaborating on the information you get in the Index Explain API results?

Reply:
Right so, having this enabled will add additional information when using the explain api.